app/models/user.rb

```diff
@@ -0,0 +1,30 @@
+class User < ApplicationRecord
+  has_many :tweets, dependent: :destroy
+
+  has_many :follower_tweets, -> { distinct }, through: :followers, source: :tweets
+  has_many :following_tweets, -> { distinct }, through: :followings, source: :tweets
+
+  has_many :follower_ids, class_name: 'UserFollowing', foreign_key: 'following_id', dependent: :destroy, inverse_of: :follower, counter_cache: :followers_count
```

Comment:
let's name this something else. A method named follower_ids should return ids and not the records

Reply:
changed to `follower_references`